README.md

```diff
@@ -120,9 +119,61 @@ FLAGS = {
 
 The set of conditions can be none (flag will never be enabled), one (only condition that has to be met for the flag to be enabled), or many (all have to be met for the flag to be enabled).
 
-Additional conditions can be added in the Django or Wagtail admin for any defined flag (illustrated in [Usage](#usage)). Conditions added in the Django or Wagtail admin can be changed without restarting Django, conditions defined in `settings.py` cannot.
+Additional conditions can be added in the Django or Wagtail admin for any defined flag (illustrated in [Usage](#usage)). Conditions added in the Django or Wagtail admin can be changed without restarting Django, conditions defined in `settings.py` cannot. See below [for a list of built-in conditions](#built-in-conditions).
+
+### Using flags in code
+
+Flags can be used in Python code:
+
+```python
+from flags.state import flag_enabled
+
+if flag_enabled('MY_FLAG', request=a_request):
+	print("My feature flag is enabled")	
+```
+
+Django templates:
+
+```django
+{% flag_enabled 'MY_FLAG' as my_flag %}
+{% if my_flag %}
+  <div>
+    I’m the result of a feature flag.   
+  </div>
+{% endif %}
+```
+
+Jinja2 templates:
+
+```jinja
+{% if flag_enabled('MY_FLAG', request) %}
+  <div>
+    I’m the result of a feature flag.   
+  </div>
+{% endif %}
+```
+
+And Django `urls.py`:
+
+```python
+from flags.urls import flagged_url, flagged_urls
+
+urlpatterns = [
+    flagged_url('MY_FLAG', r'^an-url$', view_requiring_flag, state=True),
+]
+```
+
+See the [API documentation below](#api) for more details and examples.
+
+### Using flags in Wagtail pages
 
-#### Built-in conditions
+The most recent draft of a Wagtail page can also be revealed using a feature flag if the page either inherits from `FlaggablePage` or `FlaggablePageMixin`. First the flag [must be defined in settings](#defining-flags). Then in the particular page's editor interface there should be a "Feature flags" panel that will allow you to serve the latest draft revision of the page rather than the latest published revision if a particular feature flag is enabled. 
+
+![Configuring page drafts based on feature flag state](https://raw.githubusercontent.com/cfpb/wagtail-flags/master/screenshot_page.png)
```

Comment:
This link 404's for me.

Reply:
Yeah, it's going to until this merges into master.

It's an absolute URL so that the image works when we upload the package to pypi.